Src/DependencyCollector/Shared/Implementation/HttpProcessing.cs

```diff
@@ -217,9 +217,10 @@ internal object OnBegin(object thisObj, bool skipIfNotNew)
                         if (webRequest.Headers[RequestResponseHeaders.CorrelationContextHeader] == null)
                         {
 #if NET45
-                            if (Activity.Current.Baggage.Any())
+                            var currentActivity = Activity.Current;
```

Comment:
Actual fix is here, but it's important to update DiagnosticSource to the one published to nuget to make sure everything works well together